public/404.html

```diff
@@ -1,12 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    /*define char same all page*/
     <meta charset="utf-8">
     <title>Hoodie</title>
-
+    /*internal css*/
     <link rel="stylesheet" type="text/css" href="style.css">
   </head>
   <body>
+    /*bigest char*/
```

Comment:
note that `/* */` are JavaScript comments :) For HTML comments you do `<!-- -->` :) 

And in this case, I think the comments are not needed, the HTML code is very simple already, don’t you think?

Reply:
Sorry.I'm forget